example/hello_classification_node/main.js

```diff
@@ -195,92 +144,99 @@ async function main() {
     warning(`Error to parse mean or std: ${error}`);
     process.exit(-1);
   }
-  const color = options.color;
 
-  if (iterations <= 0) {
-    warning('The number of iterations should be greater than 0.');
-    process.exit(-1);
-  }
   if (!(mean instanceof Array)) {
     warning('The mean is not an array.');
     process.exit(-1);
-  } else if (mean.length != 3) {
+  } else if (mean.length !== 3) {
     warning('The length of mean is not 3.');
     process.exit(-1);
   }
   if (!(std instanceof Array)) {
     warning('The std is not an array.');
     process.exit(-1);
-  } else if (std.length != 3) {
+  } else if (std.length !== 3) {
     warning('The length of std is not 3.');
     process.exit(-1);
   }
-  const preprocess = (mean[0] === 0 && mean[1] === 0 && mean[2] === 0 &&
-                      std[0] === 1 && std[1] === 1 && std[2] === 1) ?
-      false :
-      true;
+  const preprocess = !(mean[0] === 0 && mean[1] === 0 && mean[2] === 0 &&
+                       std[0]  === 1 && std[1]  === 1 && std[2]  === 1);
```

Comment:
Why add another space before `===`?

Reply:
just to align mean and std lines. I can remove these spaces

Reply:
Fixed after running `clang-format -i -style=Google `